### ISSUE TITLE
Running commitizen with cz-customizable won't commit

### ISSUE THREAD

**Issue by arthur-xavier** [leonardoanalista/cz-customizable#30]
After being prompted for all inputs, and after the confirmation of the commit, commitizen simply ends without the commit (success) response.

I'm using node v4.0.0 and npm 3.8.9 and commitizen 2.8.1

![screen shot 2016-05-10 at 2 15 34 pm](https://cloud.githubusercontent.com/assets/2164548/15155632/bde72abe-16b9-11e6-8350-dff3f98d2f17.png)

**Comment by arthur-xavier** [comment]
Changing `index.js:216` from
```js
], function(answers) {
```

to
```js
]).then(function(answers) {
```

Makes it work.

**Comment by leonardoanalista** [comment]
It is very strange. It has been working with node 4 and npm 3 for a while. It could be the comittizen version. Comittizen depends on InquireJs and maybe the api has changed to a promise. I ll check when I get a hand on my computer

**Comment by leonardoanalista** [comment]
yes, it was what I imagined. 

commitizen updated Inquirer.js from 0.12.0 to 1.0.2. That is a breaking change but release was minor:
https://github.com/commitizen/cz-cli/commit/a538dced#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R76

Feel free to sent a PR with the change you made plus unit tests. Otherwise I'll update cz-customizable tonight or as soon as I can. 

NOTE: This will be a possible breaking changes for this adapter.

Thanks for reporting.

I raised an issue with commitzen so next time we get notified about future API changes: https://github.com/commitizen/cz-cli/issues/227

cheers
Leonardo
